TUTORIAL.md

```diff
@@ -142,7 +145,9 @@ cd server
 ./mvnw spring-boot:run
 ```
 
-Navigate to <http://localhost:8080> and you'll be prompted to login.
+Navigate to <http://localhost:8080> and you'll be prompted to log in. 
+
+// log in vs login = verb vs noun, aka, we log in on the login page
```

Comment:
You can fix on this branch, or I can do in my "tech-review" branch. Probably best that you make the change and delete the comment on line 150.